packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/Camera.java

```diff
@@ -122,6 +126,12 @@ public void onOrientationChanged(int i) {
         new CameraZoom(
             characteristics.get(CameraCharacteristics.SENSOR_INFO_ACTIVE_ARRAY_SIZE),
             characteristics.get(CameraCharacteristics.SCALER_AVAILABLE_MAX_DIGITAL_ZOOM));
+    supportedImageFormats = new HashMap<>();
+    // Current supported outputs
+    {
+      supportedImageFormats.put("yuv420", 35);
+      supportedImageFormats.put("jpeg", 256);
+    };
```

Comment:
This should be avoided, since we are now overwriting a static variable with instance data. If you pick one of the above suggestions initialising the `supportedImageFormats` statically, this code can be removed.